File: benchmark/benchmark.py
import time
import tinysegmenter

segmenter = tinysegmenter.TinySegmenter()
f = open('timemachineu8j.txt')
text = f.read()

try:
    text = text.decode('utf-8')
except:
    pass

start = time.time()
for _ in range(100):
    segmenter.segment(text)
elapsed_time = time.time() - start
print ("elapsed_time:{0}".format(elapsed_time)) + "[sec]"

File: benchmark/benchmark.pl
use tinysegmenter;
use Encode;

open my $fh, '<', 'timemachineu8j.txt';
my $text = decode_utf8(do { local $/; <$fh> });

my $start = time;
for (1..100) {
    tinysegmenter->segment($text);
}
my $end = time;

printf "%d sec\n", $end-$start;
